aws/templates/solution/solution_ecs.ftl

```diff
@@ -53,6 +53,47 @@
             name=formatComponentLogGroupName(tier, component) /]
     [/#if]
 
+    [#if deploymentSubsetRequired("efs", true) && 
+            ecs.ClusterWideStorage?has_content && ecs.ClusterWideStorage ]
+
+        [#assign ecsEFSVolumeId = formatEFSId(tier, component)]
+        [#assign ecsEFSVolumeName = formatName( tier, component, "efs")]
+        [#assign ecsEFSSecurityGroupId = formatComponentSecurityGroupId( tier, component,"efs")]
+        [#assign ecsEFSIngressSecurityGroupId = formatDependentSecurityGroupIngressId(ecsEFSSecurityGroupId) ]
+
+        [@createComponentSecurityGroup
+            mode=listMode
+            tier=tier
+            component=component 
+            extensions="efs"
```

Comment:
not sure we need "efs" extension - more for when we use getOccurrences

Reply:
Since the security group is created as part of the ECS creation the component details are for the ECS instance instead of EFS. Since this security group is for access to the EFS instance created as part of the ECS component I added it, otherwise the group name would have matched the ecs cluster not the efs instance

Reply:
ok - makes sense